apps/api/src/routers/admin.py

```diff
@@ -210,6 +210,49 @@ async def confirm_attendance() -> None:
         )
 
 
+@router.get(
+    "/attending",
+    dependencies=[
+        Depends(require_role([Role.DIRECTOR, Role.ORGANIZER, Role.VOLUNTEER]))
+    ],
+)
+async def attending() -> list[dict[str, object]]:
+    records = await mongodb_handler.retrieve(
+        Collection.USERS,
+        {"role": Role.APPLICANT, "status": Status.ATTENDING},
+        [
+            "_id",
+            "status",
+            "role",
+            "application_data.first_name",
+            "application_data.last_name",
+        ],
+    )
+
+    return records
+
+
+@router.post("/checkin/{uid}")
+async def checkin(
+    uid: str,
+    associate: User = Depends(
+        require_role([Role.DIRECTOR, Role.ORGANIZER, Role.VOLUNTEER])
+    ),
+) -> None:
+    new_checkin_entry = {"uid": associate.uid, "time": datetime.now()}
```

Comment:
Issue: I think MongoDB will require a zoned time, we can use `utc_now` similar to as in `submit_review`.
Suggestion: given such a basic structure, could we just use a tuple rather than a dict?